src/panels/lovelace/entity-rows/hui-cover-entity-row.ts

```diff
@@ -0,0 +1,82 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+
+import "../components/hui-generic-entity-row.js";
+import "../../../components/ha-cover-controls.js";
+import "../../../components/ha-cover-tilt-controls.js";
+
+import { isTiltOnly } from "../../../util/cover-model.js";
+import { HomeAssistant } from "../../../types.js";
+import { EntityRow, EntityConfig } from "./types.js";
+
+class HuiCoverEntityRow extends LitElement implements EntityRow {
+  public hass?: HomeAssistant;
+  private _config?: EntityConfig;
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      _config: {},
+    };
+  }
+
+  public setConfig(config: EntityConfig): void {
+    if (!config) {
```

Comment:
We should make sure that we only allow entities of the cover domain

Reply:
Isn't that the job of `create-row-element.js`?

Reply:
Good point 👍 forgot we're in an entity row.